ts/src/bybit.ts

```diff
@@ -3790,9 +3791,8 @@ export default class bybit extends Exchange {
         if (isLimit) {
             request['price'] = this.priceToPrecision (symbol, price);
         }
-        const exchangeSpecificParam = this.safeString (params, 'time_in_force');
         const timeInForce = this.safeStringLower (params, 'timeInForce');
-        const postOnly = this.isPostOnly (isMarket, exchangeSpecificParam === 'PostOnly', params);
+        const postOnly = this.isPostOnly (isMarket, timeInForce === 'PostOnly', params);
```

Comment:
@ttodua did you also check `createSpotOrder`, `createUnifiedMargin` and `createUSDCOrder` ?

Reply:
@ttodua  ⬆️

Reply:
sorry forgot to handle them, switching to it now.

Reply:
completed now.